src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBranch.java

```diff
@@ -291,5 +299,6 @@ public void setMinZ(boolean dc, double lowImpedanceThreshold) {
         if (piModel.setMinZ(lowImpedanceThreshold, dc)) {
             LOGGER.trace("Branch {} has a low impedance, set to min {}", getId(), lowImpedanceThreshold);
         }
+        this.zeroImpedanceBranch = false;
```

Comment:
I think I need it, because this is done after the branch has been conctructed, so we have to override the boolean that defines if it is a zeroi impedance branch or not.